### PR TITLE
Learner scorer

### PR DESCRIPTION
**Pull request by kernc**
Updates. :kissing:

### CODE REVIEW

**Comment by kernc** [comment]
Updated with RF Regression scorer. If you agree to these changes and merge them, I have no further reservations with merging https://github.com/biolab/orange3/pull/774 as well. :sunflower:

File: Orange/widgets/classify/owrandomforest.py [hunk 1]
```diff
@@ -19,14 +19,14 @@ class OWRandomForest(widget.OWWidget):
...
+    outputs = [("Learner", RandomForestLearner),
                ("Model", RandomForestClassifier)]
 
     want_main_area = False
     resizing_enabled = False
 
     LEARNER = RandomForestLearner
-    learner_name = settings.Setting("RF Classification Learner")
+    learner_name = settings.Setting("RF Classifier")
     n_estimators = settings.Setting(10)
```
Comment (VesnaT) [inline Orange/widgets/classify/owrandomforest.py:30]: These names are not consistent. Normally we name model Classifier (not learner).

File: Orange/tests/test_random_forest.py [hunk 2]
```diff
@@ -68,15 +68,25 @@ def test_predict_numpy_reg(self):
...
+        self.assertEqual(['petal length', 'petal width'],
+                         sorted([data.domain.attributes[i].name
+                                 for i in np.argsort(scores)[-2:]]))
+
+        data = Table('housing')
+        learner = RandomForestRegressionLearner()
+        scores = learner.score_data(data)
+        self.assertEqual(['LSTAT', 'RM'],
+                         sorted([data.domain.attributes[i].name
+                                 for i in np.argsort(scores)[-2:]]))
```
Comment (VesnaT) [inline Orange/tests/test_random_forest.py:87]: Please create a new method for regression part.

File: Orange/tests/test_random_forest.py [hunk 3]
```diff
@@ -68,15 +68,25 @@ def test_predict_numpy_reg(self):
...
 
     def test_scorer_feature(self):
         np.random.seed(42)
-        data = Table('test4.tab')
+        data = Table('iris')
```
Comment (VesnaT) [inline Orange/tests/test_random_forest.py]: What is wrong with test4.tab?

File: Orange/widgets/classify/owrandomforest.py (hunk 1 above)
Comment (kernc) [inline Orange/widgets/classify/owrandomforest.py:30]: Well, these names are too long to display nicely in the narrow table headers. How about `'RF Class. Learner'` like you had before? Or perhaps the table needs patching to left-align the headers.

File: Orange/tests/test_random_forest.py (hunk 2 above)
Comment (kernc) [inline Orange/tests/test_random_forest.py:87]: Good call.

File: Orange/tests/test_random_forest.py (hunk 3 above)
Comment (kernc) [inline Orange/tests/test_random_forest.py]: I don't remember. I think when I looked at test4.tab, I thought it too trivial a file to test the scorer more thoroughly. Will take back.

**Comment by kernc** [comment]
Updated according to comments.

**Pull request merged**